ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -127,6 +128,27 @@ patternsMethod
    | REGEX
    ;
 
+fillnullCommand
+   : FILLNULL (fillNullWithTheSameValue
+   | fillNullWithFieldVariousValues)
+   ;
+
+fillNullWithTheSameValue
+   : WITH nullReplacement IN nullableField (COMMA nullableField)*
+   ;
+
+fillNullWithFieldVariousValues
+   : USING nullableField EQUAL nullReplacement (COMMA nullableField EQUAL nullReplacement)*
+   ;
+
+nullableField
```

Comment:
not sure the purpose for `nullableField` and `nullReplacement`, except for explicit trees.  But 🤷 .

Reply:
then we could use things like `fieldList` instead of `nullableField (COMMA nullableField)*`
maybe something like:

```
fillNullWithTheSameValue
   : WITH nullReplacement = valueExpression IN nullableFieldList = fieldList
   ;
```

Reply:
and 
```
fillNullWithFieldVariousValues
   : USING nullReplacementExpression (COMMA nullReplacementExpression)*
   ;

nullReplacementExpression
   : nullableField = fieldExpression EQUAL nullReplacement = valueExpression
```

Reply:
Is there a reason we're using `valueExpression` instead of `expression`?

Reply:
We want an expression that produces a single value. `expression` could produce a column such as `x + 5`.

Reply:
True.  @YANG-DB do you have a preference?  Should we raise an issue and tag the issue to be fixed in https://github.com/opensearch-project/piped-processing-language/issues/23?

Reply:
@normanj-bitquill @acarbonetto 
I would prefer going with the 3rd option (`Fix the syntax here and create a PR for opensearch-spark with a similar change`)
thanks

Reply:
Updated here and created a PR for the Spark project.

https://github.com/opensearch-project/opensearch-spark/pull/968